modules/transactionpool/transactionpool.go

```diff
@@ -121,7 +121,10 @@ func (tp *TransactionPool) FeeEstimation() (min, max types.Currency) {
 	// TODO: The fee estimation tool should look at the recent blocks and use
 	// them to guage what sort of fee should be required, as opposed to just
 	// guessing blindly.
-	return types.SiacoinPrecision.Mul64(10).Div64(1e3), types.SiacoinPrecision.Mul64(25).Div64(1e3)
+	//
+	// TODO: The current minimum has been reduced significantly to account for
+	// legacy renters that are not correctly adding transaction fees.
+	return types.SiacoinPrecision.Mul64(1).Div64(1e3), types.SiacoinPrecision.Mul64(25).Div64(1e3)
```

Comment:
Need to change comment for `FeeEstimation` in modules/transactionpool.go (line 79) to reflect new minimum